src/main/cpp/subsystems/Chassis.cpp

```diff
@@ -15,9 +15,6 @@ std::shared_ptr<Chassis> Chassis::getInstance() {
   return self;
 }
 
-std::shared_ptr<NetworkTable> table = nt::NetworkTableInstance::GetDefault().GetTable("limelight");
```

Comment:
table was moved to `Chassis.hpp`. 

```
#include "networktables/NetworkTable.h"
#include "networktables/NetworkTableInstance.h"
```
Also needs to be moved. Also use `<>` for system includes.

Reply:
Are both of these needed in the `.h`? Perhaps only `#include "networktables/NetworkTable.h"` needs to move?